libfwupdplugin/fu-pefile-firmware.c

```diff
@@ -80,6 +81,8 @@ fu_pefile_firmware_parse_section(FuFirmware *firmware,
 	/* create new firmware */
 	if (g_strcmp0(sect_id, ".sbom") == 0) {
 		img = fu_coswid_firmware_new();
+	} else if (g_strcmp0(sect_id, ".sbatlevel") == 0) {
+		img = fu_sbatlevel_section_firmware_new();
```

Comment:
i think this is a bit verbose. Maybe fu_sbatlevel_firmware_new() or fu_sbatlevel_section_new() perhaps?

Reply:
The `.sbatlevel` section contains 2 sbatlevel. I will call it `fu_sbatlevel_section_new`.